test/unit/dash.DashAdapter.js

```diff
@@ -675,12 +678,11 @@ describe('DashAdapter', function () {
 
                     expect(mediaInfoArray[0].audioChannelConfiguration).to.be.instanceOf(Array);
                     expect(mediaInfoArray[0].audioChannelConfiguration.length).equals(2);
-                    expect(mediaInfoArray[0].audioChannelConfiguration[0]).equals('6');
-
-                    expect(mediaInfoArray[0].audioChannelConfigurationsWithSchemeIdUri).to.be.instanceOf(Array);
-                    expect(mediaInfoArray[0].audioChannelConfigurationsWithSchemeIdUri.length).equals(2);
-                    expect(mediaInfoArray[0].audioChannelConfigurationsWithSchemeIdUri[0]).to.be.instanceOf(DescriptorType);
-                    expect(mediaInfoArray[0].audioChannelConfigurationsWithSchemeIdUri[1].value).equals('0xF801');
+                    expect(mediaInfoArray[0].audioChannelConfiguration[0]).to.be.instanceOf(DescriptorType);
+                    expect(mediaInfoArray[0].audioChannelConfiguration[0].schemeIdUri).equals('urn:mpeg:mpegB:cicp:ChannelConfiguration');
```

Comment:
Suggest to move `urn:mpeg:mpegB:cicp:ChannelConfiguration` to `Constants.js`

Reply:
See comment above, should only be replaced outside of unit tests

Reply:
Again:
Using constants.js for Unit-tests?
I'm rather hesitant to do so.

Reply:
Sorry, I missed to update the page.